.vs/config/applicationhost.config

```diff
@@ -137,8 +137,8 @@
             <add name="Clr2IntegratedAppPool" managedRuntimeVersion="v2.0" managedPipelineMode="Integrated" CLRConfigFile="%IIS_USER_HOME%\config\aspnet.config" autoStart="true" />
             <add name="Clr2ClassicAppPool" managedRuntimeVersion="v2.0" managedPipelineMode="Classic" CLRConfigFile="%IIS_USER_HOME%\config\aspnet.config" autoStart="true" />
             <add name="UnmanagedClassicAppPool" managedRuntimeVersion="" managedPipelineMode="Classic" autoStart="true" />
-            <applicationPoolDefaults managedRuntimeLoader="v4.0" >
-                <processModel/>
+            <applicationPoolDefaults managedRuntimeLoader="v4.0">
```

Comment:
Can you tell me how to do that. I see an answer like this, http://stackoverflow.com/questions/12481639/remove-files-from-git-commit, but I've never used git reset and would be flying blind with it.